content/zh-cn/docs/concepts/cluster-administration/_index.md

```diff
@@ -32,32 +31,40 @@ It assumes some familiarity with core Kubernetes [concepts](/docs/concepts/).
 <!--
 ## Planning a cluster
 
-See the guides in [Setup](/docs/setup/) for examples of how to plan, set up, and configure Kubernetes clusters. The solutions listed in this article are called *distros*.
-
-Not all distros are actively maintained. Choose distros which have been tested with a recent version of Kubernetes.
-
-Before choosing a guide, here are some considerations:
+See the guides in [Setup](/docs/setup/) for examples of how to plan, set up, and configure
+Kubernetes clusters. The solutions listed in this article are called *distros*.
 -->
 ## 规划集群   {#planning-a-cluster}
 
 查阅[安装](/zh-cn/docs/setup/)中的指导，获取如何规划、建立以及配置 Kubernetes
-集群的示例。本文所列的文章称为*发行版* 。
+集群的示例。本文所列的文章称为**发行版**。
```

Comment:
*distros*

Reply:
Oh, I followed the [style guide](https://kubernetes.io/zh-cn/docs/contribute/style/style-guide/#%E7%94%A8%E7%B2%97%E4%BD%93%E5%AD%97%E8%A1%A8%E7%8E%B0%E7%94%A8%E6%88%B7%E7%95%8C%E9%9D%A2%E5%85%83%E7%B4%A0) to replace italic with bold when localizing to the zh-cn text. Need a confirmation about this style change.